src/Microsoft.Dnx.Tooling/Building/BuildContext.cs

```diff
@@ -144,13 +144,30 @@ public void PopulateDependencies(PackageBuilder packageBuilder)
 
         public void AddLibs(PackageBuilder packageBuilder, string pattern)
         {
+            AddLibs(packageBuilder, pattern, false);
+        }
+
+        public void AddLibs(PackageBuilder packageBuilder, string pattern, bool recursiveSearch)
+        {
+            var searchOption = SearchOption.TopDirectoryOnly;
+            if (recursiveSearch)
+            {
+                searchOption = SearchOption.AllDirectories;
+            }
             // Add everything in the output folder to the lib path
-            foreach (var path in Directory.EnumerateFiles(_outputPath, pattern))
```

Comment:
Target location and file patterns are same as before. The only pattern that is added is searching for assembly names that end with `.resources` which is searched in all folders under the `_outputPath`